hack/cherry-pick.sh

```diff
@@ -36,9 +36,11 @@ fi
 
 pushd "${UPSTREAM_REPO_LOCATION}" > /dev/null
 os::build::require_clean_tree
-git fetch
 
-selector="$(os::build::commit_range $1 origin/master)"
+remote="$(git remote -v | grep kubernetes/kubernetes | awk '{print $1}' | uniq)"
```

Comment:
Cherry pick has to work on any upstream.  Just take an env var that modifiers the remote.